### ISSUE TITLE
1080 Snowboarding(USA/Jap)

### ISSUE THREAD

**Issue by FreyBentos** [Rinnegatamante/DaedalusX64-vitaGL-Compatibility#109]
# **Game summary**
- Game name: 1080 Snowboarding
- Game version: US/Jap

# **Daedalus X64 summary**
- Version: v0.2

# **Issues**
- Many texture issues, in menua and in game.
- Some items don't load at all and menu runs very slow around 10-13 fps
- Sky box flickers and doesn't render properly.
- Some models have issues with warping 
 where parts of the model are being stretched into the distance, similar to 
 WWF wrestlemania/no mercy.
- In game works but plays slow fps is stated as 20 but it feels much slower, feels like its playing at 5- 
 10 fps
-some snow effects not rendering correctly.

# Screenshots

# **Recommended labels**
- Slow
- In-Game ( high )

**Comment by Sarkie** [comment]
@FreyBentos does the game cover load for you?

**Comment by Sarkie** [comment]
@Rinnegatamante are the any oddities with this entry, the cover doesn't load and the CRC matchesn and the labels load.

Latest vpk and data zip

**Comment by Rinnegatamante** [comment]
Check if the filename of the cover is correct in roms.ini

**Comment by Sarkie** [comment]
Yeah it's the same. I'm not sure why it isn't loading. Only thing I can think it's junk line endings or something

**Comment by Dreamer1972** [comment]
Using Dx64 v0.6 nightly (ba53a2)

Settings:  Dynarec / Legacy or modern rend 

Everything appears to tender correct.  Sound has a bit of crackle, sync audio rate fixes this...but audio gets pitchy.
Legacy rend seems a bit faster. About 25fps with audio.  Audio disabled gives about 28fps.  Game freezes on black screen when saving best times.  But times are saved.  Overall...playable but slow.

**Comment by ozymandis12** [comment]
Gameplay is perfect and normal speed if on modern render (euro version)

**Comment by adrianbass360** [comment]
EUROPE VERSION PLAYABLE. Runs fine with in-game BGM disable and mono sound. You can use this config and latest daedalusx64 version

[Configs.zip](https://github.com/user-attachments/files/19817297/Configs.zip)
[DaedalusX64 v0.6d (f6ccfb)(02-15-25).zip](https://github.com/user-attachments/files/19817296/DaedalusX64.v0.6d.f6ccfb.02-15-25.zip)
